.github/workflows/main.yml

```diff
@@ -204,6 +204,78 @@ jobs:
         dotnet pack ./src/Neo.Cryptography.BLS12_381 \
         --configuration Release \
         --output ./out
+    - name : Pack (ApplicationLogs)
+      if: steps.check_tag.outputs.statusCode == '404'
+      run: |
+        dotnet pack ./src/Plugins/ApplicationLogs \
+        --configuration Release \
+        --output ./out
```

Comment:
Can easily be moved under matrix strategy in order to avoid code duplication between the `Pack (*)` steps and run them in parallel.

Reply:
Ref. https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy.

Reply:
yes i know. I will be fixing all these workflows.